libraries/abstractions/pkcs11/test/iot_test_pkcs11_globals.h

```diff
@@ -31,11 +31,11 @@ extern CK_SESSION_HANDLE xGlobalSession;
 extern CK_FUNCTION_LIST_PTR pxGlobalFunctionList;
 extern CK_SLOT_ID xGlobalSlotId;
 extern CK_MECHANISM_TYPE xMechanismType;
```

Comment:
Would it be better to rename xMechanismTyep and xKey to also include global in name?

Reply:
I'll rename these as well